core/src/App.html

```diff
@@ -1,4 +1,4 @@
-<svelte:window on:resize="{onResize}"/>
+<svelte:window on:resize="{onResize}" />
```

Comment:
checked other files and we have no space at the end of the tag. Not a thing, but maybe it is better to be consistent.